routing/cross_mwm_index_graph.hpp

```diff
@@ -44,10 +44,21 @@ uint32_t constexpr GetFeaturesOffset() noexcept
 }
 
 template <>
-uint32_t constexpr GetFeaturesOffset<connector::TransitId>() noexcept
+uint32_t constexpr GetFeaturesOffset<TransitId>() noexcept
 {
   return FakeFeatureIds::kTransitGraphFeaturesStart;
 }
+
+template <typename CrossMwmId>
+inline void CheckIfConnectorIsFound(NumMwmId neighbor, bool isConnectorFound)
```

Comment:
Do we need inline for templates?

Reply:
No. Not at all.

Reply:
However, we need. Otherwise we get a linkage error:
```
[ 98%] Linking CXX executable ../desktop.app/Contents/MacOS/desktop
duplicate symbol __ZN7routing9connector22AssertConnectorIsFoundINS0_9TransitIdEEEvtb in:
    ../libmap.a(routing_manager.cpp.o)
    ../libmap.a(framework.cpp.o)
duplicate symbol __ZN7routing9connector22AssertConnectorIsFoundINS0_9TransitIdEEEvtb in:
    ../libmap.a(routing_manager.cpp.o)
    ../librouting.a(cross_mwm_graph.cpp.o)
duplicate symbol __ZN7routing9connector22AssertConnectorIsFoundINS0_9TransitIdEEEvtb in:
    ../libmap.a(routing_manager.cpp.o)
    ../librouting.a(index_router.cpp.o)
duplicate symbol __ZN7routing9connector22AssertConnectorIsFoundINS0_9TransitIdEEEvtb in:
    ../libmap.a(routing_manager.cpp.o)
    ../librouting.a(single_vehicle_world_graph.cpp.o)
duplicate symbol __ZN7routing9connector22AssertConnectorIsFoundINS0_9TransitIdEEEvtb in:
    ../libmap.a(routing_manager.cpp.o)
    ../librouting.a(transit_world_graph.cpp.o)
ld: 5 duplicate symbols for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[3]: *** [desktop.app/Contents/MacOS/desktop] Error 1
make[2]: *** [qt/CMakeFiles/desktop.dir/all] Error 2
make[1]: *** [qt/CMakeFiles/desktop.dir/rule] Error 2
make: *** [desktop] Error 2
```

Reply:
You're right. We need `inline` keyword for specialization only.